docs/advanced-topics/stack-monitoring.asciidoc

```diff
@@ -0,0 +1,150 @@
+:page_id: stack-monitoring
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= Stack Monitoring
+
+You can enable link:https://www.elastic.co/guide/en/elasticsearch/reference/current/monitor-elasticsearch-cluster.html[Stack Monitoring]
+on your Elasticsearch and Kibana to collect and ship their metrics and logs.
+
+You just have to reference the separate monitoring Elasticsearch cluster in the `spec.monitoring` section.
+
+[source,yaml,subs="attributes,callouts"]
+----
+cat $$<<$$EOF | kubectl apply -f -
+apiVersion: elasticsearch.k8s.elastic.co/{eck_crd_version}
+kind: Elasticsearch
+metadata:
+  name: monitored-sample
+  namespace: production
+spec:
+  version: {version}
+  monitoring:
+    metrics:
+      elasticsearchRefs:
+        - name: monitoring
+          namespace: observability
+    logs:
+      elasticsearchRefs:
+        - name: monitoring
+          namespace: observability
+  nodeSets:
+  - name: default
+    count: 1
+    config:
+      node.store.allow_mmap: false
+---
+apiVersion: kibana.k8s.elastic.co/{eck_crd_version}
+kind: Kibana
+metadata:
+  name: monitored-sample
+  namespace: production
+spec:
+  version: {version}
+  elasticsearchRef:
+    name: monitored-sample
+    namespace: production
+  monitoring:
+    metrics:
+      elasticsearchRefs:
+        - name: monitoring
+          namespace: observability
+    logs:
+      elasticsearchRefs:
+        - name: monitoring
+          namespace: observability
+  count: 1
+EOF
+----
+
+The use of `namespace` is optional if the Elasticsearch cluster and the Kibana are running in the same namespace.
```

Comment:
Should this be a [callout ](https://github.com/elastic/docs#callouts) in the YAML sample?

Reply:
Good idea.